src/components/Visualization.tsx

```diff
@@ -243,28 +261,32 @@ const Visualization = () => {
         defaultSize="500px"
         minSize="150px"
       >
-        <div className="reactflow-wrapper" data-testid="react-flow-wrapper">
-          <ReactFlow
-            nodes={nodes}
-            edges={edges}
-            defaultViewport={defaultViewport.current}
-            edgeTypes={edgeTypes}
-            nodeTypes={nodeTypes}
-            onDragOver={onDragOver}
-            onNodeClick={onNodeClick}
-            onNodesChange={onNodesChange}
-            onEdgesChange={onEdgesChange}
-            snapToGrid
-            snapGrid={[15, 15]}
-            deleteKeyCode={null}
-            zoomOnDoubleClick={false}
-            className="panelCustom"
-          >
-            {/*<MiniMap nodeBorderRadius={2} className={'visualization__minimap'} />*/}
-            <VisualizationControls />
-            <Background color="#aaa" gap={16} />
-          </ReactFlow>
-        </div>
+        {visibleFlowsInformation.isCanvasEmpty ? (
```

Comment:
OMG, I forgot to add tests for this clause, I'll add this test case and merge it afterward.

@tplevko covering this condition at the integration test side would be enough or should we add an e2e as well?

Reply:
Here we have [the missing test](https://github.com/KaotoIO/kaoto-ui/pull/1967/files#diff-86c560251a1c2da7c959dcf452624515f2e2ca08ddfbced394c24f58425b0a07R79)

Reply:
Well from my POV, I think the test you added should be enough, can add more testing for this in e2e if requested.